modules/azure/github-actions-terraform-setup/buildingblock/resources.github.tf

```diff
@@ -0,0 +1,108 @@
+# note: this building block is expected to be executed with a config.tf file as output by the "backplane" module in the
+# parent dir, this needs to provided in the BB execution enviornment
```

Comment:
no longer correct and a notice like this would make more sense in the main.tf file